store_test.go

```diff
@@ -6,16 +6,23 @@ import (
 	"testing"
 
 	"github.com/containers/storage/pkg/idtools"
+	"github.com/containers/storage/pkg/reexec"
 	digest "github.com/opencontainers/go-digest"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func newTestStore(t *testing.T, options StoreOptions) Store {
+	store, err := GetStore(options)
+	require.NoError(t, err)
+	return store
+}
+
 func TestStore(t *testing.T) {
 	wd := t.TempDir()
 
 	pullOpts := map[string]string{"Test1": "test1", "Test2": "test2"}
-	store, err := GetStore(StoreOptions{
+	storeOptions := StoreOptions{
```

Comment:
(A personal style nit: Inlining a single-purpose options structure directly seems a tiny bit cleaner to me than adding a new single-use variable. It’s more similar to keyword parameters in languages like Python, and the reader does not have to look for any other uses of the variable. Opinions do differ on this.)